src/model/UpsetColumn.ts

```diff
@@ -51,11 +51,13 @@ export class CustomSortCalculation {
 export default class UpsetColumn extends ValueColumn<number[] > {
   private sortCriteria;
   private datalength;
+  private setBinary;
 
   constructor(id: string, desc: any) {
     super(id, desc);
     this.sortCriteria = (<any>desc).sort || 'min';
-    this.datalength = (<any>desc.datalength);
+    this.datalength = (<any>desc.dataLength);
+    this.setBinary = 1;
```

Comment:
`setBinary` is a confusing name, since `setBinary` sounds like a setter for the property `binary`. + if it always `1` it should be static constant